docroot/modules/custom/prisoner_hub_prison_context/prisoner_hub_prison_context.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Prisoner Hub Prison Context'
+type: module
+description: 'Provides a Drupal service for loading in the current prison context.'
```

Comment:
Just thinking about non-Drupal developers who may end up poking around the modules, could we expand a bit on what this does and why it's useful in the description?

Reply:
Yeah good point.  This description was created before I really knew how we were going to implement it.  I can definitely be a bit more specific about it now.